hal/armv7m/cpu.c

```diff
@@ -315,8 +315,7 @@ void hal_cleanDCache(ptr_t start, size_t len)
 	(void)len;
 
 #ifdef CPU_IMXRT
-	/* Clean the whole cache, no harm in being overzealous */
-	_imxrt_cleanDCache();
+	_imxrt_cleaninvalDCacheAddr((void *)start, len);
```

Comment:
nit: this change removes `_imxrt_cleanDCache();`,  and add `_imxrt_cleaninvalDCacheAddr((void *)start, len);`, so the above
```
	(void)start;
	(void)len;
```
can be moved to `#else` block where arguments are really unused.

Reply:
Oops, thanks